app/constants.ts

```diff
@@ -0,0 +1 @@
+export const linkStyles = 'text-primary underline hover:no-underline';
```

Comment:
If you want that every link should have these styles, then just put it in the global css and with that you don't need to put it in every `Link`